packages/internal/src/__tests__/paths.test.ts

```diff
@@ -16,22 +16,27 @@ describe('paths', () => {
       )
 
       const pages = processPagesDir(pagesDir)
+      expect(pages[0].importName).toEqual('adminEditUserPage')
```

Comment:
Checking this as well was exactly what I was thinking! 👍 

Only thing I'm thinking is if it should start with a capital A or not. Like, it's a component, and those usually start with capital letters. But I'm not sure if it matters or not. What do you think?

Also, now that we're not just looking at the paths, maybe change the name of the test as well? "it accurately finds and names the pages", or something like that?

Reply:
I thought about the capital A as well, and left it as lowercase as the folder is lowercase in:
 __fixtures__/example-todo-main/web/src/pages/admin/

I was thinking it could be a future pr to allow for lowercase folder names and return uppercase importNames.  My priority was getting the bug fix in here.  Thoughts?

As for the test name, I agree, I'll fix that tonight.

Reply:
OK, made some time and made the change.

Reply:
I had a look through the code, and it doesn't look like we're using the name for more than comparing two pages with each other. And in that case upper-/lowercase doesn't matter. So I agree, let's leave it for a future PR if needed.